src/test/java/com/google/api/codegen/testsrc/protoannotations/library.proto

```diff
@@ -84,6 +84,12 @@ service LibraryService {
     option (google.api.method_signature) = "";
   }
 
+  // Lists shelves.
```

Comment:
This is to have a paginated method with `map<>` resource in it. ListShelves is a regular paginated method with repeated resource in it.

Reply:
Could you add that to the comment?

Reply:
Please lets not do it. All of the comments in the proto become the generated docs in the generated clients.

Reply:
And the problem with that is what? More changes to the baseline files? Anything else?

How about if you create the comment separated from the proto by a newline? I'd be happy with that, as it would explain things for someone reading the .proto, and maybe that wouldn't change the baseline files?

```

// ListAggregatedShelves tests paged calls that return maps of lists paged in parallel.

//ListsShelves
rpc ListAggregatedShelves.(...) returns (...) {
  ...
}

```

Reply:
Added the comment. You are right, if there is a blank line it is not recognized as the method comment.
On a side note this is not what is done i the rest of the library.proto. All the documentation/comments ther is API-specific, not gapic-generator specific.